packages/@ngtools/webpack/src/plugin.ts

```diff
@@ -210,16 +215,24 @@ export class AotPlugin implements Tapable {
           return callback();
         }
 
-        this.done.then(() => {
-          result.resource = this.skipCodeGeneration ? this.basePath : this.genDir;
-          result.recursive = true;
-          result.dependencies.forEach((d: any) => d.critical = false);
-          result.resolveDependencies = createResolveDependenciesFromContextMap(
-            (_: any, cb: any) => cb(null, this._lazyRoutes));
-
+        // there is no way to find the original request, so try to
```

Comment:
You can use a special property set by the function above, that should work, right?